README.md

```diff
@@ -15,6 +15,8 @@
 
 ![](https://user-images.githubusercontent.com/5351546/222356748-14276998-501b-4d2a-9b09-b8cff3018204.png)
 
+OpenDAL is an Apache Software Foundation project. Learn more at [opendal.incubator.apache.org](https://opendal.incubator.apache.org/).
```

Comment:
BTW, I don't think we need another section here for `o.a.o`.

Reply:
Thanks!